modules/acs-cloud-requirements.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acscs-default-requirements.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-requirements_{context}"]
+= General requirements for RHACS Cloud Service
+
+[role="_abstract"]
+{product-title-managed} has some system requirements that must be met before you can install it.
+
+[WARNING]
+====
+You must not install {product-title-managed-short} on:
+
+* Amazon Elastic File System (Amazon EFS). Use the Amazon Elastic Block Store (Amazon EBS) with the default *gp2* volume type instead.
+* Older CPUs that do not have the Streaming SIMD Extensions (SSE) 4.2 instruction set.
+For example, Intel processors older than _Sandy Bridge_ and AMD processors older than _Bulldozer_.
+(These processors were released in 2011.)
+====
+
+To install {product-title-managed-short}, you must have one of the following systems:
+
+* {ocp} version {ocp-supported-version} or later, and cluster nodes with a supported operating system of {op-system-first} or {op-system-base-full}.
+* a supported managed Kubernetes platform, and cluster nodes with a supported operating system of Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
++
+For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix].
+
+Cluster nodes minimum requirements:
+
```

Comment:
Consider making the lead-in wording more clear, perhaps by saying something like, "The following minimum requirements apply to cluster nodes:" 

The [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) refers to this structure as more conducive to ease of translation: "**Translation consideration**: To avoid possible translation problems, use complete sentences to introduce lists and not phrases."

Reply:
+1. As a bonus, it would be parallel in construction to "To install RHACS Cloud Service, you must have one of the following systems:"